packages/ihe-gateway-sdk/src/common/shared.ts

```diff
@@ -0,0 +1,12 @@
+const OID_REGEX = /^(urn:oid:)?([0-9]+(\.[0-9]+)*)$/;
+
+export function normalizeOid(oid: string | undefined): string {
+  if (!oid) throw new Error("OID must be present");
```

Comment:
Why do we accept `undefined` it its required?

Reply:
very true